packages/graphql/tests/tck/issues/1760.test.ts

```diff
@@ -125,48 +125,48 @@ describe("https://github.com/neo4j/graphql/issues/1760", () => {
         expect(formatCypher(result.cypher)).toMatchInlineSnapshot(`
             "MATCH (this:\`ApplicationVariant\`)
             WHERE this.current = $param0
-            WITH *,  apoc.cypher.runFirstColumnSingle(\\"MATCH (this)<-[:HAS_BASE]-(n:BaseObject) RETURN n.id\\", {this: this, auth: $auth}) AS relatedId
```

Comment:
From what I can tell, this PR has broken a key performance improvement for a customer, where we evaluate the sorting of a custom Cypher field at the _beginning_ of execution. In this test you can see that this has once again been moved down to the end of the Cypher query.

Reply:
Good point. I'll write a quick perf. benchmark to see if I can reproduce the problem